### PR TITLE
os: automate setup of a ubuntu host from scrach

### PR DESCRIPTION
**Pull request by hunchback**
provide a means for setting up an ubuntu host from scrach (works on VM and on WSL), addressing part of https://github.com/IBM/the-mesh-for-data/issues/127

### CODE REVIEW

File: .mk/verify.mk [hunk 1]
```diff
@@ -18,13 +18,6 @@ define license_python
 	($(ABSTOOLBIN)/license_finder --python-version=3 || true)
 endef
 
-GO_VERSION:=1.13
```
Comment (roee88) [inline .mk/verify.mk:21]: Why is this removed?

File: go.mod [hunk 2]
```diff
@@ -3,19 +3,22 @@ module github.com/ibm/the-mesh-for-data
...
 
 require (
+	cloud.google.com/go v0.66.0 // indirect
 	emperror.dev/errors v0.7.0
 	github.com/go-chi/chi v4.1.2+incompatible
 	github.com/go-chi/render v1.0.1
 	github.com/go-logr/logr v0.1.0
 	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.2
```
Comment (roee88) [inline go.mod:12]: Incompatible with out project

**Review by roee88 — CHANGES_REQUESTED** [review]
Please open an issue before submitting a PR, especially for things that were not previously discussed in an issue or elsewhere. Also, please follow the contribution instructions for signing off commits.

File: .mk/verify.mk (hunk 1 above)
Comment (hunchback) [reply .mk/verify.mk:21]: there are new version of go 1.14 and 1.15, thus no need to limit or bind our project to any specific version (and leave go version and IDE choice to developer).

File: go.mod (hunk 2 above)
Comment (hunchback) [reply go.mod:12]: reverting ... but sooner or later it will have to be addressed as currently go.mod automatically suggests this change when building the project

**Comment by roee88** [comment]
See my comment on the issue.

Technical comments:
- Have you actually tested the project with go 1.14/1.15? 
- Missing go install
- Missing jdk install
- Missing docker post-install for linux instructions (e.g., creating group)
- Missing other tools such as make and jq
- Why are conntrack, ethtool and apache2-utils needed?

Rather than trying to solve this for "any ubuntu VM" I'm closing this PR and we can discuss in the issue to see if we really want to vendor such files (I don't think so).
